dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -15,11 +15,11 @@ internal static class SemanticFunctionConstants
 6. 'input' does not need to be specified if it consumes the 'output' of the previous function.
 7. To save an 'output' from a <function>, to pass into a future <function>, use <function.{FunctionName} ... setContextVariable: ""$<UNIQUE_VARIABLE_KEY>""/>
 8. To save an 'output' from a <function>, to return as part of a plan result, use <function.{FunctionName} ... appendToResult: ""RESULT__$<UNIQUE_RESULT_KEY>""/>
-9. Append an ""END"" XML comment at the end of the plan.
+9. You can only use ""<if>"" or ""<else>"" nest else tags, don't use elseif.
```

Comment:
Actually this `nested` as extra and is not needed in the prompt, will remove.

When I asked to don't use elseif, it got it right. is nesting the ifs and elses as a result.

```
<if>
<else>
   <if></if>
   <else></else> (nested else)
</if>
<else></else>
```
Before that Rule it was hallucinating an `<else if>` tag, which is actually smart but was not valid Xml and the runtime would not parse. 

I could resort to LLM to Parse it but to avoid extra complexity I will skip the potential `elseif` tag